build/build.go

```diff
@@ -575,7 +575,7 @@ func NewMappingCallback(m *sourcemap.Map, goroot, gopath string) func(generatedL
 	}
 }
 
-func jsFilesFromDir(dir string) ([]string, error) {
+func JsFilesFromDir(dir string) ([]string, error) {
```

Comment:
Now that this identifier is exported, it should have a doc string that describes what it does.

Also, according to Go style, the acronym should have the same case, so it's `JSFilesFromDir`, not `JsFilesFromDir`.